main/scripts/docs-map-generator.js

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs-extra");
+
+module.exports = OsmAndDocMapGenerator;
+
+const PATH = "./docs/";
+const URL = "https://osmand.net/docs/";
```

Comment:
Chicken & egg problem, we need a running website to get a structure...

Reply:
CC @alex-osm  - I'm not sure this is the right approach.

Reply:
Agree, why do we need osmand.net hardcoded here?